src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -128,20 +104,16 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
     ) {
       applyThemesOnElement(this, this.hass.themes, this._config.theme);
     }
-
-    if (changedProps.has("hass")) {
-      toggleAttribute(this, "rtl", computeRTL(this.hass!));
-    }
   }
 
   protected render(): TemplateResult {
     if (!this._config || !this.hass) {
       return html``;
     }
 
-    const stateObj = this.hass.states[this._config.entity];
+    const stateObj = this.hass.states[this._config.entity] as WeatherEntity;
 
-    if (!stateObj) {
+    if (!stateObj || stateObj.state === UNAVAILABLE) {
```

Comment:
What is the new plan for unavailable? This card doesn't have an input so it doesn't really fall under the same as the light card, etc

Reply:
Just print that it is unavailable